src/main/java/frc/robot/commands/IntakeFromSource.java

```diff
@@ -72,11 +86,12 @@ public void end(boolean interrupted) {
     subTransfer.setTransferNeutralOutput();
     subShooter.setDesiredVelocities(lastDesiredSpeedLeft, lastDesiredSpeedRight);
     subPitch.setPitchAngle(lastDesiredPitch, subClimber.collidesWithPitch());
+
```

Comment:
end is missing some things here. (ex. Reposition game piece)
Please update your branch with main and refer to IntakeGroundGamePiece for what's missing